examples/nextjs-blog/pages/authors/profile.js

```diff
@@ -1,12 +1,44 @@
-const Profile = function(props) {
-  return <h1>{props.fullName}</h1>
+import { flamelinkApp as app } from '../../utils/flamelink'
+import PropTypes from 'prop-types'
+
+const Profile = function(props = {}) {
+  const { user } = props
+
+  if (!user) {
+    return 'No user data'
```

Comment:
if you want to be sexy (and you are) you can 

`
import { Router } from '../../routes';

...

Router.pushRoute('/404') // this will end up using error.js page
`
or create a component

`
import Error from 'next/error'

export const throw404 = () => {
  if (process.browser) {
    return <Error statusCode={404} />
  }
  const e = new Error()
  e.code = 'ENOENT'
  throw e
}
`

Reply:
sweet, thanks @gitdubz